CoreScriptsRoot/Modules/BackpackScript3D.lua

```diff
@@ -499,4 +500,15 @@ function BackpackPanel:OnMouseLeave(x, y)
 	EnableHotbarInput(false)
 end
 
+local VRHub = require(RobloxGui.Modules.VR.VRHub)
+VRHub.ModuleOpened.Event:connect(function(moduleName)
+	local module = VRHub.RegisteredModules[moduleName]
```

Comment:
I would like to more a function for this `GetRegisteredModuleByName` or something so that other modules don't know how we are storing modules in VRHub